src/sampleData/sampleBook.js

```diff
@@ -0,0 +1,68 @@
+const sampleBook = {
+  "rank": 2,
+  "rank_last_week": 1,
+  "weeks_on_list": 49,
+  "asterisk": 0,
+  "dagger": 0,
+  "primary_isbn10": "038554734X",
+  "primary_isbn13": "9780385547345",
+  "publisher": "Doubleday",
+  "description": "A scientist and single mother living in California in the 1960s becomes a star on a TV cooking show.",
+  "price": "0.00",
+  "title": "LESSONS IN CHEMISTRY",
+  "author": "Bonnie Garmus",
+  "contributor": "by Bonnie Garmus",
+  "contributor_note": "",
+  "book_image": "https://storage.googleapis.com/du-prd/books/images/9780385547345.jpg",
+  "book_image_width": 328,
+  "book_image_height": 500,
+  "amazon_product_url": "https://www.amazon.com/dp/038554734X?tag=NYTBSREV-20",
+  "age_group": "",
+  "book_review_link": "",
+  "first_chapter_link": "",
+  "sunday_review_link": "",
+  "article_chapter_link": "",
+  "isbns": [
+      {
+          "isbn10": "038554734X",
+          "isbn13": "9780385547345"
+      },
+      {
+          "isbn10": "0593507533",
+          "isbn13": "9780593507537"
+      },
+      {
+          "isbn10": "0385547374",
+          "isbn13": "9780385547376"
+      }
+  ],
+  "buy_links": [
+      {
+          "name": "Amazon",
+          "url": "https://www.amazon.com/dp/038554734X?tag=NYTBSREV-20"
+      },
+      {
+          "name": "Apple Books",
+          "url": "https://goto.applebooks.apple/9780385547345?at=10lIEQ"
+      },
+      {
+          "name": "Barnes and Noble",
+          "url": "https://www.anrdoezrs.net/click-7990613-11819508?url=https%3A%2F%2Fwww.barnesandnoble.com%2Fw%2F%3Fean%3D9780385547345"
+      },
+      {
+          "name": "Books-A-Million",
+          "url": "https://du-gae-books-dot-nyt-du-prd.appspot.com/redirect?url1=https%3A%2F%2Fwww.anrdoezrs.net%2Fclick-7990613-35140%3Furl%3Dhttps%253A%252F%252Fwww.booksamillion.com%252Fp%252FLESSONS%252BIN%252BCHEMISTRY%252FBonnie%252BGarmus%252F9780385547345&url2=https%3A%2F%2Fwww.anrdoezrs.net%2Fclick-7990613-35140%3Furl%3Dhttps%253A%252F%252Fwww.booksamillion.com%252Fsearch%253Fquery%253DLESSONS%252BIN%252BCHEMISTRY%252BBonnie%252BGarmus"
+      },
+      {
+          "name": "Bookshop",
+          "url": "https://du-gae-books-dot-nyt-du-prd.appspot.com/redirect?url1=https%3A%2F%2Fbookshop.org%2Fa%2F3546%2F9780385547345&url2=https%3A%2F%2Fbookshop.org%2Fbooks%3Faffiliate%3D3546%26keywords%3DLESSONS%2BIN%2BCHEMISTRY"
+      },
+      {
+          "name": "IndieBound",
+          "url": "https://du-gae-books-dot-nyt-du-prd.appspot.com/redirect?url1=https%3A%2F%2Fwww.indiebound.org%2Fbook%2F9780385547345%3Faff%3DNYT&url2=https%3A%2F%2Fwww.indiebound.org%2Fsearch%2Fbook%3Fkeys%3DLESSONS%2BIN%2BCHEMISTRY%2BBonnie%2BGarmus%26aff%3DNYT"
+      }
+  ],
+  "book_uri": "nyt://book/717e0ead-9782-567d-af27-d9a6cfb1856f"
+}
+
+export default sampleBook;
```

Comment:
The structure of the BookInfo component is relying on the selectedBook prop to be a single object